plugins/default/lib/openshift.py

```diff
@@ -27,3 +27,13 @@ def _exec_in_pods(oc, project, pods, cmd):
 
 def exec_in_pods(project, pods, cmd):
     return _exec_in_pods(oc, project, pods, cmd)
+
+
+def get_project():
+    # Manually get the namespace via workaround because of a bug in 'oc project', see:
+    # <insert bug url here>
+    # return openshift.oc("project", "-q").rstrip("\n")
```

Comment:
We should definitely open the bug, but not sure we need this here. The other approach is a documented way to call the API from a pod:
http://kubernetes.io/docs/user-guide/accessing-the-cluster/#accessing-the-api-from-a-pod